client/disputes/utils.ts

```diff
@@ -45,3 +49,8 @@ export const isDueWithin = ( { dueBy, days }: IsDueWithinProps ): boolean => {
 	const isPastDue = now.isAfter( dueByMoment );
 	return isWithinDays && ! isPastDue;
 };
+
+export const isInquiry = ( dispute: Dispute | CachedDispute ): boolean => {
+	// Inquiry disputes statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
```

Comment:
Super-minor nitpick:
```suggestion
	// Inquiry dispute statuses are one of `warning_needs_response`, `warning_under_review` or `warning_closed`.
```